# Patch to use PNG which is not enabled by default
# Contribution from the Crick HPC team
# uploaded by J. Sassmannshausen
diff -Nru cytosim-3ecab2584958081f0bbb42642e849c84f7d34c0b-orig/makefile.inc cytosim-3ecab2584958081f0bbb42642e849c84f7d34c0b/makefile.inc
--- cytosim-3ecab2584958081f0bbb42642e849c84f7d34c0b-orig/makefile.inc	2019-01-17 09:14:25.000000000 +0000
+++ cytosim-3ecab2584958081f0bbb42642e849c84f7d34c0b/makefile.inc	2019-03-13 11:11:44.649196000 +0000
@@ -38,7 +38,7 @@
 #     1 : macport installation
 #     2 : homebrew installation
 
-HAS_PNG := 0
+HAS_PNG := 1
 
 #-------------------------------------------------------------------------------
 #---------------------------  Platform Detection  ------------------------------
@@ -155,7 +153,7 @@
 
     ifneq ($(HAS_PNG), 0)
 
-        LIB_PNG := $(USRLIB)/libpng.a $(USRLIB)/libz.a
+        LIB_PNG := -L$(USRLIB) -lpng 
         ING_PNG :=
 
     endif
